backend/lib/endpoints/schema/reports.js

```diff
@@ -1,15 +1,38 @@
 const S = require("fluent-schema");
 const { strictSchema } = require("./utils");
+const { POST_STATUS } = require("../../models/Post");
+const { MODERATOR_ACTIONS } = require("../../models/Audit");
 
 const createPostReportSchema = {
-  body: strictSchema().prop("reason", S.string().required()),
+  body: strictSchema().prop("reason", S.string().required().maxLength(500)),
+  params: strictSchema().prop("postId", S.string().required()),
 };
```

Comment:
it's better to add `description` and `response` codes for new endpoints, refer to @jessica-lau 's [commit for example](https://github.com/FightPandemics/FightPandemics/pull/2049/commits/131184bb19b490a8781b4bae69133dfa44a5fad7) 😁

Reply:
She already did in that commit, for `createPostReportSchema` https://github.com/FightPandemics/FightPandemics/pull/2049/commits/131184bb19b490a8781b4bae69133dfa44a5fad7#diff-3fbedfbc2f534dc85376a9c486d68b39bf969d0766b6914e409607ebd84d134d
the other endpoints are internal and should not be documented (?)
`getPostReportsSchema`, `getAuditLogSchema`, `moderatorActionSchema`